src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,22 @@
+import styles from "./Breadcrumb.module.css"
+import { BreadcrumbItem, LinkProps } from "./BreadcrumbItem"
+
+interface Props {
+  links: LinkProps[]
```

Comment:
Instead of reusing `LinkProps` here and exposing `isCurrent`—an implementation detail—to the caller, I would suggest creating a separate `Link` interface that purely represents what the caller needs to know about a link (omitting `isCurrent`).

Then, in your `LinkProps` (or `BreadcrumItemProps`), you could make `isCurrent` required. This way we eliminate confusion at the caller level, and reduce the ability for misusing `BreadcrumbItem` when someone is adjusting the implementation internally.